README.md

```diff
@@ -16,7 +16,7 @@ which are not actively developed anymore.
 ### Our goals
 
 - [Goals for the next release](https://github.com/htm-community/nupic.cpp/blob/master/RELEASE_V1_GOALS.md)
-- Actively developed C++ core library for HTM/nupic.core (Numenta's repos are in maintanance mode only)
+- Actively developed C++ core library for nupic.core (Numenta's repos are in maintanance mode only)
```

Comment:
might be good to put a link to that repo?

Reply:
Will be fixed in PR #526